packages/core/src/common/styles.ts

```diff
@@ -39,6 +39,7 @@ export interface Theme {
     headerFontStyle: string;
     baseFontStyle: string;
     fontFamily: string;
+    fontSize: string;
```

Comment:
`editorFontSize` would be a better name here as it may be unclear what the intent of this theme property is otherwise.

Reply:
yep makes sense.